SQL_practice.sql

```diff
@@ -55,13 +54,13 @@ select P.province_name,
   ( select count(*) from
 	     (select province_id, count(city) from vendors group by city having count(city) is 1)
 	                     group by province_id
-			  ) as 'Number on unique cities in province'
+			  ) as 'n_unique_cities'
```

Comment:
Please try to avoid only one letter, e.g. not n_unique_cities, but number_unique_cities